modules/api/src/main/java/net/countercraft/movecraft/craft/Craft.java

```diff
@@ -376,8 +373,13 @@ public int getTickCooldown() {
      * gets the speed of a craft in blocks per second.
      * @return the speed of the craft
      */
-    public double getSpeed(){
-        return 20*(type.getCruiseSkipBlocks()+1)/(double)getTickCooldown();
+    public double getSpeed() {
+        if(cruiseDirection == 0x42 || cruiseDirection == 0x43) {
```

Comment:
This is needed to correctly adjust the speed display based on which skipBlocks is being used.